wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -63,6 +63,42 @@ sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
     }
   }
 
+  override def sendToAddresses(
+      addresses: Vector[BitcoinAddress],
+      amounts: Vector[CurrencyUnit],
+      feeRate: FeeUnit,
+      fromAccount: AccountDb): Future[Transaction] = {
```

Comment:
No `markAsReserved`?

Reply:
It is defaulted to false

Reply:
I guess what I mean is why can't it be passed into this API? Don't we want to mark the utxo as reserved possibly?